src/guides/v2.3/graphql/tutorials/checkout/checkout-billing-address.md

```diff
@@ -25,53 +25,42 @@ Use the [setBillingAddressOnCart]({{ page.baseurl }}/graphql/mutations/set-billi
 
 ## Add a new billing address
 
-The following mutation adds a new billing address. `{ CART_ID }` is the unique shopping cart ID from [Step 2. Create empty cart]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-add-product-to-cart.html).
+Similar to the previous step, we will add a billing address to the customer. `{ CART_ID }` is the unique shopping cart ID from [Step 2. Create empty cart]({{ page.baseurl }}/graphql/tutorials/checkout/checkout-add-product-to-cart.html). In this example, the `default_billing` parameter is set to `true`. The street address is also different, so we can see that different addresses are being created.
```

Comment:
Get rid of everything but adding a billing address to the cart, so the procedure is the same for guests and customers. 

The shipping and billing addresses are now different. That's OK, but be sure to mention that.